addons/mail/static/src/models/rtc_call_participant_card.js

```diff
@@ -104,6 +119,27 @@ registerModel({
                 return this.invitedGuest.name;
             }
         },
+        /**
+         * @private
+         * @returns {string|FieldCommand}
+         */
+        _computeOutboundConnectionTypeText() {
+            if (!this.rtcSession || !this.rtcSession.localCandidateType) {
+                return clear();
+            }
+            return _.str.sprintf(
+                this.env._t('Outbound: %s (%s)'),
```

Comment:
Better use sprintf with named arguments, in order to provide context for translations and to allow the substitution strings to be reordered.

\+ double quotes